apps/bridge-dapp/src/components/Header/WalletButton.tsx

```diff
@@ -133,15 +133,18 @@ export const WalletButton: FC<{ account: Account; wallet: WalletConfig }> = ({
     }, [] as string[]);
 
     // Download the notes as a file
-    downloadString(JSON.stringify(notes), 'notes.json', '.json');
+    downloadString(JSON.stringify(notes), `notes-${Date.now()}.json`, '.json');
   }, [allNotes, notificationApi]);
 
   // TODO: Implement a function when user click on the new notes link
   // on the notification
-  const handleNewNotes = useCallback(async (notes: Note[]) => {
-    logger.info('Handle new notes: ', notes);
-    logger.warn('New notes function is not implemented yet');
-  }, []);
+  const handleNewNotes = useCallback(
+    async (notes: Note[]) => {
+      logger.info('Handle new notes: ', notes);
```

Comment:
@AtelyPham will `logger.info` spill into the console? Is this log intentional?

Reply:
Yes, maybe it can spill into the console, this is for development only. I'll remove it